packages/embark/src/lib/modules/blockchain_listener/index.js

```diff
@@ -22,15 +24,40 @@ class BlockchainListener {
     this.events = embark.events;
     this.logger = embark.logger;
     this.ipc = ipc;
+    this.isDev = this.embark.config.env === "development";
```

Comment:
Do we have a constant for "development"? Would be nice to get rid of concrete strings in the code